aiosmtpd/controller.py

```diff
@@ -71,9 +71,9 @@ def start(self):
     def _stop(self):
         self.loop.stop()
         try:
-            _all_tasks = asyncio.Task.all_tasks
-        except AttributeError:   # pragma: skipif_lt_py39
             _all_tasks = asyncio.all_tasks
+        except AttributeError:   # pragma: skipif_gt_py36
+            _all_tasks = asyncio.Task.all_tasks
```

Comment:
Apparently Python 3.8 raises `DeprecationWarning` for access to asyncio.Task.all_tasks, messing up the captured warnings inside pytest test cases.

So we invert the attribute access.